lua/lazyvim/plugins/extras/lang/yaml.lua

```diff
@@ -0,0 +1,30 @@
+return {
+
+  -- add yaml specific modules to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "yaml" })
+      end
+    end,
+  },
+
+  -- correctly setup lspconfig
+  {
+    "neovim/nvim-lspconfig",
+    dependencies = {
+      "someone-stole-my-name/yaml-companion.nvim",
```

Comment:
Is this really required for those who don't need K8s integration?

Reply:
removed